libs/ui-lib/lib/ocm/components/Routes.tsx

```diff
@@ -18,9 +18,9 @@ export const Routes: React.FC<{ allEnabledFeatures: FeatureListType }> = ({
     <Provider store={storeDay1}>
       <AssistedUILibVersion />
       <Switch>
```

Comment:
While using CompatRouter and changing everyting into v6 Switch is no longer in use. You can use <Routes> component